### DISCUSSION TITLE
to make the frog picture depend on the weather.

### DISCUSSION THREAD

**Discussion by mmleko21** [PranshulGG/WeatherMaster#104] [Ideas]
Hi! 
First, thank u for this app! 
This is what I have been looking for a long time - weather with my favorite frog, independent of google. 
The app is great, the forecast is accurate, but the pictures with the frog are shown randomly. 
Please, make the pictures correspond to the weather, for example:
it's snowing - frog in a hat on the background of snowfall
sunny and hot - frog sunbathing

now everything is mixed up and does not correspond to the weather.

**Comment by PranshulGG** [comment]
> Hi! First, thank u for this app! This is what I have been looking for a long time - weather with my favorite frog, independent of google. The app is great, the forecast is accurate, but the pictures with the frog are shown randomly. Please, make the pictures correspond to the weather, for example: it's snowing - frog in a hat on the background of snowfall sunny and hot - frog sunbathing
> 
> now everything is mixed up and does not correspond to the weather.

@mmleko21 Thank you for your kind words! As for the froggy images, they aren't random; they change based on the weather conditions. There’s just a wider variety of images now. Let me know if anything seems off!

**Reply by mmleko21** [reply]
Thank u! Now I see it, I was confused that it was cold outside and the picture was summer weather. 
Now the forecast has changed to snowfall and the picture is also snowing, everything is just as it should be!
Btw, I want to help with the Russian translation, but PoEditor write:
"This project can't accept new contributors at this time. Please try again later"
And in the app some things are not translated. Can I translate it? 
For example:
![1000028717-edited](https://github.com/user-attachments/assets/6155a4e4-2a64-48c0-b105-e980a3ed85d5)
Its my best weather app, real, thank u so much!

**Reply by PranshulGG** [reply]
Yeah, I’m glad you liked it! The images aren’t really based on whether it’s summer or winter, but rather on the conditions. As for translation, POEditor has some issues, but it will be available on Weblate soon.
